onnxruntime/core/providers/cpu/tensor/gather_elements.cc

```diff
@@ -298,18 +230,10 @@ Status GatherElements::Compute(OpKernelContext* context) const {
   if (indices_shape.Size() == 0)
     return Status::OK();
 
-  auto* ttp = context->GetOperatorThreadPool();
```

Comment:
I guess there is nothing to be done even if `input_data_shape.Size()` == 0 as well - bail out early ?

Reply:
ValidateInputShapes should take care of that, as it'd be an error to have input data not being at least as big as the index data.